File: Week_04/FindContentChildren_455.py
class FindContentChildren_455:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        gi = 0
        si = 0
        while gi < len(g) and si < len(s) :
            if  g[gi] <= s[si] :
                gi+=1
            si+=1
        return gi

            
        

File: Week_04/FindContentChildren_455.java
class FindContentChildren_455 {
    public int findContentChildren(int[] g, int[] s) {
        Arrays.sort(g);
        Arrays.sort(s);
        int gi = 0;
        int si = 0;
        while ( si < s.length && gi < g.length ){
            if ( g[gi] <= s[si] ){
                gi++;
            }
            si++;
        }
        return gi;
    }
}